waiter/src/waiter/scheduler.clj

```diff
@@ -727,7 +731,7 @@
           (let [duration (metrics/duration-between start-time current-time)]
             (metrics/report-duration waiter-schedule-timer duration)
             (metrics/report-duration service-schedule-timer duration)
-            #_(statsd/timer! service-schedule-timer duration)))
+            (statsd/timing! metric-group "schedule_time" duration)))
```

Comment:
Don't the `statsd` calls only need to be triggered on the leader?

Reply:
Yes! I'm removing this for now. This isn't trivial, and this PR is already complex enough without this.